pkgs/development/libraries/xdg-desktop-portal-wlr/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, fetchFromGitHub
+, meson, ninja, pkgconfig, wayland-protocols
+, pipewire, wayland, elogind, systemd, libdrm }:
+
+stdenv.mkDerivation rec {
+  pname = "xdg-desktop-portal-wlr";
+  version = "0.1.0";
+
+  src = fetchFromGitHub {
+    owner = "emersion";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "12k92h9dmn1fyn8nzxk69cyv0gnb7g9gj7a66mw5dcl5zqnl07nc";
+  };
+
+  nativeBuildInputs = [ meson ninja pkgconfig wayland-protocols ];
+  buildInputs = [ pipewire wayland elogind systemd libdrm ];
```

Comment:
`elogind` is an alternative to `libsystemd` if the distribution doesn't use systemd, I don't think this dependency is used here.

Reply:
That's true, good catch! You can even see it [here](https://github.com/emersion/xdg-desktop-portal-wlr/blob/cf3e02695e3d54de142b7b1b9740fb2776be1fc4/include/xdpw.h#L5-L11), I'll try to keep that in mind in my next PRs

Reply:
That begs the question: is nix on non-systemd Linux distributions supported? Should with have an argument `withSystemd ? true` so that you can use it outside of NixOS and systemd distros?

Reply:
> is nix on non-systemd Linux distributions supported?

Nix is used by people on other distros that don't have systems. For example with home-manager. So it can be a consideration to have. If it's too much work I wouldn't hold the package back just for that. People on other systems are probably more likely to install xdg-desktop-portal-wlr using their system package manager instead.